darwin/torch/transforms.py

```diff
@@ -378,7 +399,13 @@ def _post_process(self, albumentation_output: dict, annotation: dict) -> tuple:
             else:
                 output_annotation["masks"] = torch.stack(masks)
             if "area" in annotation:
-                output_annotation["area"] = torch.sum(output_annotation["masks"], dim=[1, 2])
+                output_annotation["area"] = torch.sum(
+                    output_annotation["masks"], dim=[1, 2]
+                )
+        elif "masks" in annotation:
```

Comment:
what is this checking for/why is it needed?

Reply:
We needed to remove the masks tag to get around an albumentation bug, but we add it back again with an empty list if there are no annotations. This is so any code on the user side that assumes there to be a mask key wont break. (It might break for empty masks, but at least not a keyerror).